Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/DriverInformation.hs

```diff
@@ -75,6 +76,34 @@ data DriverInformationE e = DriverInformation
   }
   deriving (Generic)
 
+data DriverSummary = DriverSummary
+  { totalEarnings :: Money,
+    bonusEarned :: Money,
+    totalCompletedTrips :: Int,
+    lateNightTrips :: Int,
+    lastRegistered :: UTCTime
+  }
+  deriving (Generic, ToJSON, FromJSON, ToSchema, Show)
+
+data DriverMissedOpp = DriverMissedOpp
+  { cancellationRate :: Int,
+    ridesCancelled :: Maybe Int,
```

Comment:
Why maybe?

Reply:
In driverStats , cancelled count was maybe and I am just using that count